src/openloco/objects/wall_object.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "../localisation/stringmgr.h"
```

Comment:
`string_id` is defined in `types.hpp` nowadays, which `stringmgr.h` includes. Including `types.hpp` directly instead should make for quicker compilation times.